ClavaLaraApi/src-lara-clava/clava/clava/graphs/cfg/CfgBuilder.js

```diff
@@ -61,6 +61,11 @@ class CfgBuilder {
    */
   #nextNodes;
 
+  /**
+   * Creates a new instance of the CfgBuilder class
+   * @param {joinpoint} $jp 
+   * @param {boolean} deterministicIds If true, uses deterministic ids for the graph ids (e.g. id_0, id_1...). Otherwise, uses $jp.astId whenever possible
```

Comment:
Signal in the documentation that the parameter is optional, and that has a default value. The format that is being used is:

@param {boolean} [deterministicIds = false]